### DISCUSSION TITLE
/oo/interface.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#51] [Announcements]
# /oo/interface.html

Java程序员进阶之路，小白的零基础Java教程，认真聊聊Java接口

https://tobebetterjavaer.com/oo/interface.html

**Comment by DylanLJC** [comment]
希望增加一个标注的功能，看的时候会想把重点的东西记录上

**Comment by yeahQing** [comment]
希望可以增加一个选中某一句话可以对这句话写批注，就像飞书文档那种。
